.env

```diff
@@ -0,0 +1,3 @@
+QANAPI_BIND=127.0.0.1:9001
+QANAPI_DSN=clickhouse://127.0.0.1:9000?debug=true&database=pmm&x-multi-statement=true
+GO111MODULE=on
```

Comment:
I think we (as a team) need to make a decision about using Go modules before using them.